designs/virtualization/4.6/empty-states/empty-states.md

```diff
@@ -0,0 +1,54 @@
+# Virtualization Default Empty States
+Foday Kargbo - UXD
+## Background
+Virtualization is being introduced to the console. Virtualization will be accessed via the main left navigation panel. Once the user navigates to Virtualization, they will see 2 tabs, one for the Virtual Machines screen, and one for the Virtual Machine Templates screen. The Virtual Machines tab will be selected by default.
+## Objectives
+Users who have not created virtual machines or virtual machine templates will not be able to see any data on these screens. 
+TBD: Do we want to create an empty state screen for the virtualization dashboard?
+
+The goals of this task:
+- Create empty state screens when there is no data to show
+- Inform the user about the current state
+- Educate the user about what they will see when data is available
+- Guide the user to take action
+- Explore any useful content that might help the user be contextually productive?
+## 1. Virtual machine default empty state screen
+![Virtual machines empty state screen](img/virtual-machines-empty-state.png "Virtual machines empty state screen")
+## UI Inventory
+1. Tabs
+2. Graphic (Icon)
+3. Title text
+4. Body text
+5. Call to action button
+6. Secondary call to action (optional)
+
+## Discussion
+1. Tabs: Are we ok with the title on this tab? Yes (Resolved)
+2. Graphic: We should use an image with context 
+3. Title text: Are we ok with this title?
+4. Body text: Are we ok with this text?
+5. Call to action button: On click/tap it opens a dropdown menu with 3 options:
+    - Create via option 1
+    - Create via option 2
+    - Create via option 3
+6. Secondary call to action: (optional) Is there a need for this? We could use this as an opportunity to educate the user. Alternatively, we could use this to present a secondary option when creating a virtual machine.
```

Comment:
I think we really want/need to push quick starts here. We could even have a more prominent presentation of them if the user hasn't created anything yet. 
With this new push to provide images for an OS source when the operator ships we could either point users to create a custom VM OR create from a template. It could be worth exploring what it looks like on this page to have both.

Reply:
Ok, I'll explore some directions for how we could present quick starts to the use here.

Reply:
I've added two screens showing possible placements for Quick Starts and Create from Template.